web/satellite/src/components/apiKeys/ApiKeysItem.vue

```diff
@@ -34,11 +36,11 @@ export default class ApiKeysItem extends Vue {
     @Prop({default: () => new ApiKey('', '', '', '')})
     private readonly itemData: ApiKey;
```

Comment:
i would prefer apiKey instead of itemData

Reply:
its common prop for Vlist, so its name is standardized